pkg/manifest/manifest.go

```diff
@@ -60,6 +61,10 @@ type Manifest struct {
 	Obj *unstructured.Unstructured
 }
 
+func (r *resourceId) Equal(id *resourceId) bool {
```

Comment:
I have mixed feelings about a public method on a private type.  If this needs to be public, can we make `ResourceID` public?

Reply:
I don't see an issue. The private type merely says nothing outside the package can or needs to reference the type. 

I do need to make `Manifest.id` public though.